### PR TITLE
create a basic Linux development environment

### PR DESCRIPTION
**Pull request by deeso**

### CODE REVIEW

File: docs/development/linux.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (marpaia) [inline docs/development/linux.md]: remove newline

File: docs/development/linux.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Setting up a Linux Development Environment
+
+### Install some dependencies
+
+sudo apt-get install xzip gyp libjs-underscore libuv1-dev dep11-tools deps-tools-cli
```
Comment (marpaia) [inline docs/development/linux.md]: wrap this (and all following commands) in back ticks like:

```
sudo apt-get install xzip gyp libjs-underscore libuv1-dev dep11-tools deps-tools-cli 
```

File: docs/development/linux.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
+## Setting up a Linux Development Environment
+
+### Install some dependencies
+
+sudo apt-get install xzip gyp libjs-underscore libuv1-dev dep11-tools deps-tools-cli 
+
+### Create a temp directory, download and place the **node** and **golang** bins 
+mkdir tmp
```
Comment (marpaia) [inline docs/development/linux.md]: add a newline after the header and before this line

Comment (marpaia) [reply docs/development/linux.md]: if you can, do that below as well on the next few sections

File: docs/development/linux.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+wget https://nodejs.org/dist/v9.4.0/node-v9.4.0-linux-x64.tar.xz
+xz -d node-v9.4.0-linux-x64.tar.xz
+tar -xf node-v9.4.0-linux-x64.tar
+sudo cp -rf node-v9.4.0-linux-x64/bin /usr/local/
+sudo cp -rf node-v9.4.0-linux-x64/include /usr/local
+sudo cp -rf node-v9.4.0-linux-x64/lib /usr/local
+sudo cp -rf node-v9.4.0-linux-x64/share /usr/local
+npm install -g yarn
+
+#### install **go**
```
Comment (marpaia) [inline docs/development/linux.md]: Install `go`

File: docs/development/linux.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+### Install some dependencies
+
+sudo apt-get install xzip gyp libjs-underscore libuv1-dev dep11-tools deps-tools-cli 
+
+### Create a temp directory, download and place the **node** and **golang** bins 
+mkdir tmp
+cd tmp
+
+#### install **node** and **yarn**
```
Comment (marpaia) [inline docs/development/linux.md]: Install `node` and `yarn`

File: docs/development/linux.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+sudo cp -rf node-v9.4.0-linux-x64/lib /usr/local
+sudo cp -rf node-v9.4.0-linux-x64/share /usr/local
+npm install -g yarn
+
+#### install **go**
+wget https://dl.google.com/go/go1.9.3.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.9.3.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin:~/go/bin/
+
+#### clean-up temp directory
```
Comment (marpaia) [inline docs/development/linux.md]: Clean-up temporary directory

File: docs/development/linux.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+cd ..
+rm -rf tmp
+
+### Clone and build depenencies
+cd ~/go/src
+git clone https://github.com/kolide/fleet.git
+cd fleet
+make deps
```
Comment (marpaia) [inline docs/development/linux.md]: perhaps add some prose after this linking to the docs on using the fleet binary

**Comment by groob** [comment]
This is nice, but I'm curious what is deficient about the current linux dev environment with docker-compose for dependencies?

**Comment by deeso** [comment]
I chose to set-up the development environment outside of a `docker`, because my current deployment of `fleet` is on a virtual machine/server.

**Comment by marpaia** [comment]
It looks like the markdown is rendering incorrectly due to a formatting error.

screen shot 2018-02-14 at 3 46 14 pm

**Comment by marpaia** [comment]
Closing this due to inactivity. If you can fix the formatting issues, please re-open and we'll be happy to reconsider the PR.

**Pull request closed without merging**